packages/web/README.md

```diff
@@ -9,43 +7,61 @@ Natura Design System - Web Components
 
 ## Installation
 
-Install the packages in your project directory with:
+Install the packages in your project root directory with:
+
+### npm
 
-```sh
-# with npm
+```shell script
 npm i --save @naturacosmeticos/natds-web
+```
 
-# with yarn
+### yarn
+
+```
 yarn add @naturacosmeticos/natds-web
 ```
 
+### Minimum requirements
+
+* `react` version `16.8.4` or higher (`16.10.2` or higher recommended);
+* `react-dom` version `16.8.4` or higher (`16.10.2` or higher recommended).
+
 ## Usage
 
-```jsx highlight-line="2"
-import { Chip, Provider, themes } from '@naturacosmeticos/natds-web'
+```jsx highlight-line="3"
+import React from "react"
+import { Button, Provider, themes} from "@naturacosmeticos/natds-web"
 
 const { natura } = themes;
 
-return (
-  <Provider theme={natura.light}>
-    <Chip
-      clickable={boolean}
-      disabled={boolean}
-      label={string}
-      onDelete={function}
-    />
-  </Provider>
+export const YourApp = () => (
+   <Provider theme={natura.light}>
+      <Button color={"primary"}>Button component</Button>
+      <Button variant={"contained"}>Contained button</Button>
+   </Provider>
 )
 ```
 
-## About Fonts
+## Is included in package
+
+When you install `@naturacosmeticos/natds-web`, the following packages will be installed too:
```

Comment:
Que tal uma alteração aqui para:
Installing `@naturacosmeticos/natds-web` will also install the following packages:

Reply:
@eduardogc tem razão, vou alterar 😄